packages/jest-diff/src/printDiffs.ts

```diff
@@ -162,16 +162,18 @@ export const printAnnotation = (
     const aCount = String(changeCounts.a);
     const bCount = String(changeCounts.b);
 
-    const aPadding =
-      Math.max(bAnnotation.length - aAnnotation.length, 0) +
-      Math.max(bCount.length - aCount.length, 0);
-    const bPadding =
-      Math.max(aAnnotation.length - bAnnotation.length, 0) +
-      Math.max(aCount.length - bCount.length, 0);
-
-    // Separate annotation from count by padding plus margin of 2 spaces.
-    aRest = ' '.repeat(aPadding + 2) + aCount + ' ' + aIndicator;
-    bRest = ' '.repeat(bPadding + 2) + bCount + ' ' + bIndicator;
+    // Padding right aligns the ends of the annotations.
+    const baAnnotationLengthDiff = bAnnotation.length - aAnnotation.length;
+    const aPaddingRight = ' '.repeat(Math.max(0, baAnnotationLengthDiff));
+    const bPaddingRight = ' '.repeat(Math.max(0, -baAnnotationLengthDiff));
+
+    // Padding left aligns the ends of the counts.
+    const baCountLengthDiff = bCount.length - aCount.length;
+    const aPaddingLeft = ' '.repeat(Math.max(0, baCountLengthDiff));
+    const bPaddingLeft = ' '.repeat(Math.max(0, -baCountLengthDiff));
+
+    aRest = aPaddingRight + '  ' + aIndicator + ' ' + aPaddingLeft + aCount;
```

Comment:
Why is `right` left of `left`? 😅

Reply:
Good eyes! If you think of the indicator character as border of adjacent table cells?

Reply:
just `aIndicatorPadding`, `aCountPadding`?

Reply:
If you mean can I rename the variables to make their meaning clear, the answer is yes.